index.html

```diff
@@ -35,7 +38,7 @@ <h1 class="d-flex">
      <span class="green-text self-center center-text animate-pulse">I’m a software developer</span>
```

Comment:
- [ ] Kindly add a container with a fixed height that wraps the animated paragraph. This is because for the mobile version the animated paragraph is making all the content moves and that doesn't look good.

Reply:
> Actioned ✅  😃